ugc/api.cpp

```diff
@@ -39,23 +42,23 @@ void Api::SaveUGCOnDisk()
   m_thread.Push([this] { SaveUGCOnDiskImpl(); });
 }
 
-void Api::GetUGCImpl(FeatureID const & id, UGCCallback callback)
+void Api::GetUGCImpl(FeatureID const & id, UGCCallback const & callback)
 {
   if (!id.IsValid())
   {
-    GetPlatform().RunOnGuiThread([callback] { callback({}, {}); });
+    callback({}, {});
```

Comment:
How about to check that callback is valid?

Reply:
ok